src/analytics.ts

```diff
@@ -39,6 +39,7 @@ const _analytics = globalAny.analytics
 
 export interface AnalyticsSettings {
   writeKey: string
+  cdn?: string
```

Comment:
I believe (but correct me if I'm wrong) that this field is only useful when using AnalyticsBrowser, not AnalyticsNode. There's an `AnalyticsBrowserSettings` interface in `browser.ts` that might make more sense to add this to then - I don't think the `Analytics` class actually uses this setting at all iirc.

Reply:
Sorry, one more comment - can you add a doc tag for this as well to explain what this field is for? (We haven't been great about doing that thus far). I kind of prefer calling it `cdnEndpoint` but can be persuaded since it stays consistent with the `analytics._cdn` global.